spec/models/lockstep/user_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe 'Lockstep::User' do
+
+  context 'when calling User Invite to invite a user' do
+    it 'should return true as the email is not present already' do
+      VCR.use_cassette("models/lockstep/user/exist") do
+        response = Lockstep::User.invite(["abc@gmail.com"])
+        expect(response.first['success']).eql?(true)
```

Comment:
shouldn't this be expect(...).to eql?. You are missing the to chain which might be necessary.

Reply:
Resolved [here](https://github.com/Lockstep-Network/lockstep-sdk-ruby-rails/pull/44/commits/50147f342649efa65db02f90e35a778872d06fde)